config/foreman.hiera/tuning/common.yaml

```diff
@@ -1,4 +1,9 @@
 ---
+apache::default_mods:
+  - 'setenvif'
+  - 'alias'
+  - 'env'
```

Comment:
Rather than specifying them here, I think we should check our modules that use them and use `include apache::mod::$mod`. Ideally also set `apache::default_mods: false` in the Hiera config for the integration tests in those modules to be sure.

Reply:
Just to be explicit: https://github.com/theforeman/puppet-foreman/blob/master/spec/acceptance/hieradata/common.yaml is what I mean. The same thing for all our other modules that use Apache in acceptance tests.

The big benefit is that we don't need to maintain a list of modules that must be present in the installer. That also means users who do override it don't need to specify some magic list.

Reply:
Thank you. We define `aliases` for the reverse proxy virtual host at https://github.com/theforeman/puppet-foreman_proxy_content/blob/a3ac43579b1f456b11c989a3ef0230718a36a44e/manifests/reverse_proxy.pp#L40 so I think it's needed in both places at least

Now that we have some better tests, I'll check what issues are caused when the default Apache modules are disabled and apache::mod::alias is not yet included